airflow/providers/databricks/operators/databricks.py

```diff
@@ -150,7 +150,7 @@ class DatabricksSubmitRunOperator(BaseOperator):
     <https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsSubmit>`_
     API endpoint.
 
-    There are two ways to instantiate this operator.
+    There are three ways to instantiate this operator.
```

Comment:
This operator has 186 lines of text in docstring. That is a lot! I feel we should extract all of this information to howto guide of the operator and keep minimal description on the class code.
WDYT @josh-fell @Bowrna ?